fetch.bs

```diff
@@ -3614,83 +3673,96 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    <a for=url>origin</a> and <var>credentials</var>.
   </dl>
 
- <li><p>If <var>connection</var> is failure, return a
- <a>network error</a>.
 
- <li><p>If <var>connection</var> is not an HTTP/2 connection, <var>request</var>'s
- <a for=request>body</a> is non-null, and <var>request</var>'s <a for=request>body</a>'s
- <a for=body>source</a> is null, then <a for="header list">append</a>
- `<code>Transfer-Encoding</code>`/`<code>chunked</code>` to <var>request</var>'s
- <a for=request>header list</a>.
+  <li>
```

Comment:
Ah balls. My editor tries to be "smart" but the lack of `</li>` caused its "smarts" to be "a pain in the arse".